internal/pkg/manifest/svc.go

```diff
@@ -410,3 +410,35 @@ func ParsePortMapping(s *string) (port *string, protocol *string, err error) {
 		return nil, nil, fmt.Errorf("cannot parse port mapping from %s", *s)
 	}
 }
+
+type fromCFN struct {
+	Name *string `yaml:"from_cfn"`
+}
+
+func (e *fromCFN) isEmpty() bool {
```

Comment:
nit: 😂 why `e` for the receiver?
```suggestion
func (cfg *fromCFN) isEmpty() bool {
```